onnx/reference/ops/op_quantize_linear.py

```diff
@@ -141,7 +141,7 @@ def _run(  # noqa: PLR0911
         axis: int = 1,
         saturate: bool = True,
         block_size: int | None = None,
-        output_dtype: int | None = None,
+        output_dtype: TensorProto.DataType | None = None,
```

Comment:
This should stay int because it should be able to accept any integer and be converted to the enum.

Reply:
Unfortunately, there is no conversion from int to enum.

Reply:
Thanks. We can fix that later then